src/i18n/en/translation.json

```diff
@@ -61,6 +61,9 @@
   "button": {
     "okay": "Okay"
   },
+  "error" : {
+    "message": "We encountered a major Error. Please try reloading the page. If the problem persists, contact our support."
```

Comment:
Maybe

`Something went wrong. Please try reloading the page. If the problem persists, contact our support.`

Reply:
Yeah that sounds better